docker-compose.yml

```diff
@@ -81,7 +81,7 @@ services:
 
   mysql_integration:
     image: mysql/mysql-server:8.0
-    command: --default-authentication-plugin=mysql_native_password
+    command: --default-authentication-plugin=mysql_native_password --sql-mode="NO_ENGINE_SUBSTITUTION"
```

Comment:
The image used on CircleCI uses mysql 5.6, which seems not to have the same defaults as the image used in the dev environment.